user_test.go

```diff
@@ -60,50 +60,38 @@ func TestGetUser(t *testing.T) {
 
 func TestGetUserTeams(t *testing.T) {
 	// Arrange
-	requests := []TestRequest{
-		{
-			Request: `{"query": "query UserTeamsList($after:String!$first:Int!$user:ID!){account{user(id: $user){teams(after: $after, first: $first){nodes{alias,id},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor},totalCount}}}}",
-            "variables": {
-                {{ template "first_page_variables" }},
-                "user": "{{ template "id1"}}"
-            }
-            }`,
-			Response: `{
-                "data": {
+	testRequestOne := TestRequest{
+		Request:   `"query": "query UserTeamsList($after:String!$first:Int!$user:ID!){account{user(id: $user){teams(after: $after, first: $first){nodes{alias,id},pageInfo{hasNextPage,hasPreviousPage,startCursor,endCursor},totalCount}}}}"`,
+		Variables: `"variables": { {{ template "first_page_variables" }}, "user": "{{ template "id1"}}" }`,
+		Response: `{ "data": {
                     "account": {
                         "user": {
                             "teams": {
                                 "nodes": [
-									{{ template "teamId_1"}},
-									{{ template "teamId_2"}}
+                                  {{ template "teamId_1"}},
```

Comment:
IIRC - go fmt requires `tabs` - Edgar and I had this discussion some time ago where we tried to convert the entire codebase and couldn't figure out why everything was going back to tabs and its because `go fmt`.